services/slam/slam.go

```diff
@@ -167,3 +185,27 @@ func protobufToMappingMode(mappingMode pb.MappingMode) (MappingMode, error) {
 		return 0, errors.New("mapping mode unspecified")
 	}
 }
+
+func sensorTypeToProtobuf(sensorType SensorType) pb.SensorType {
+	switch sensorType {
+	case SensorTypeCamera:
+		return pb.SensorType_SENSOR_TYPE_CAMERA
+	case SensorTypeMovementSensor:
+		return pb.SensorType_SENSOR_TYPE_MOVEMENT_SENSOR
+	default:
+		return pb.SensorType_SENSOR_TYPE_UNSPECIFIED
+	}
+}
+
+func protobufToSensorType(sensorType pb.SensorType) (SensorType, error) {
+	switch sensorType {
+	case pb.SensorType_SENSOR_TYPE_CAMERA:
+		return SensorTypeCamera, nil
+	case pb.SensorType_SENSOR_TYPE_MOVEMENT_SENSOR:
+		return SensorTypeMovementSensor, nil
+	case pb.SensorType_SENSOR_TYPE_UNSPECIFIED:
+		fallthrough
+	default:
+		return 0, errors.New("sensor type unspecified")
+	}
+}
```

Comment:
[q] should these live in the same spot as the grpchelper functions?

Reply:
then maybe the grpchelper functions should just live in slam.go, or should be a part of the slam package? seems weird to have two different packages define helper functions

Reply:
I'll move them out to the slam package!

Reply:
Done

Reply:
we usually have all the grpc helper functions int he resource package itself. I didn't know that was not the case in slam, sorry!